model/map-jpa/src/main/resources/META-INF/auth-sessions/jpa-auth-sessions-changelog-1.xml

```diff
@@ -67,4 +67,13 @@ limitations under the License.
         </ext:createJsonIndex>
     </changeSet>
 
+    <changeSet author="keycloak" id="auth-sessions-14161">
```

Comment:
Some of our changelogs (e.g. clients) have a comment that reads `    <!-- format of id of changeSet: clients-${org.keycloak.models.map.storage.jpa.Constants.CURRENT_SCHEMA_VERSION_CLIENT} --> `

And this is not exactly following that pattern. Should we remove the comments from the other changelogs and use the issue numbers instead on every new changeset?

Reply:
yeah, first intention was to have one changeset corresponding to each schema version, then it turned out we sometimes need more than one changeset per schema version, (e.g. due to crdb deferred and initially deferrable). Now I think we should keep separate file for each schema version, like done here: https://github.com/keycloak/keycloak/pull/13183/files#diff-62b05d1598ad750b93df4c1e976e884ff107d9ade486e91bd5e216c2f4851a5aR22

When I was thinking about the id I wasn't able to come up with anything better. Do you have any suggestions? 

My take on this would be to remove the commets from all changelog files and start using issue number from now on. wdyt?

Reply:
I agree with that - it makes it easier also to track changes based on the issues that resulted in them being created.